apis/logging/v1/cluster_log_forwarder_types.go

```diff
@@ -183,10 +186,6 @@ type OutputSpec struct {
 	//   `passphrase`: Passphrase to decode an encoded TLS private key. Requires tls.key.
 	//   `ca-bundle.crt`: Custom CA to validate certificates.
 	//
-	//   `tls.insecure`: If this key is present (with any value) accept server certificates that are self-signed or have an incorrect hostname.
```

Comment:
@alanconway do we need to continue to honor this arrangement for backwards compatibility?

Reply:
@jcantrill Alan has answered this question in a separate comment. Okay to un-hold?